### ISSUE TITLE
Reacting to messages is not accessible

### ISSUE THREAD

**Issue by sukiletxe** [UnigramDev/Unigram#2763]
**Describe the bug**
When reacting, a floating window opens with some reactions and a more button. This menu cannot be focused with the keyboard, and it only can be used with object navigation. (I haven't managed to use it with Narrator, as it seems it restrict object navigation to the containing window, though I almost don't have any experience with Narrator). All controls in this window are unlabeled. The emogis cannot be read for some reason, and the more button is not labeled either (I consider this less important).

**To Reproduce**
1. Right click a message which you can react (see #2762 for a related issue on this).
2. Try pressing tab or F6. Notice the focused menu is still the messages actions menu.
3. If using NVDA, press NVDA + Numpad 8 (desktop layout) or NVDA + Shift + Up arrow (laptop layout) until hearing Unigram. Then press NVDA + Numpad 4 (desktop) or NVDA + Shift + Left arrow (laptop) until hearing "popup host" for the second time, then press NVDA + Numpad 8 (desktop) or NVDA + Shift + Down arrow (laptop) twice. Finally, press NVDA + Shift + Numpad delete (desktop) or NVDA + Shift + Backspace (laptop) to, finally, focus the reactions window. I don't have steps for Narrator, sorry.
4. Tab around. Notice that everything is unlabeled.

**Expected behavior**
The emogis should be labeled with the associated emogi symbol. As for the menu focusing, maybe a submenu inside the message actions menu (the one we can't navigate) would help. Or even a separate dialog box.

**Version Info**
 - Unigram: Unigram 8.8 (7407) X64 Direct
 - Windows: Versión 21H2 (compilación de SO 19044.1766)

**Comment by Dementor316** [comment]
It's a pity that the issue is not being fixed and continues to exist in the very latest versions of Unigram. In fact, we, as screen reader users, do not have the ability to react to any messages because it is inconvenient to access the reaction popup window and the reaction emojis themselves do not have any normal emoji icon or other convenient readable display representation.

**Comment by AlexandrKozlovskiy** [comment]
@Dementor316 +++

**Comment by Dementor316** [comment]
In the latest version of Unigram (10.3.1), it became possible to easily add certain reactions to messages using a screen reader program. To do this, simply bring up the context menu on the desired message and at the very bottom of the menu there will be a panel displaying the last 7 frequently used emojis for reactions. From this list, you can use arrow keys to select the desired reaction and press Enter/space.

However, there are still issues with the panel showing all reactions, which can be accessed by pressing "More reactions" button right away. It displays a list of all possible reactions that you can add to a message, but these reactions are not shown as emoji icons that can be easily vocalized by a screen reader program; instead they appear as names of software elements in the list, which does not convey the exact meaning of each emoji reaction and makes it impossible to understand what reactions can actually be added to a specific message.

In particular, all reactions in the panel showing all reactions are labeled as "Telegram.ViewModels.Drawers.StickerViewModel". We really need normal emoji icons displayed in this panel so that we have the ability to add correct reactions to messages using screen reader programs.

**Comment by Dementor316** [comment]
@FrayxRulez can you explain why this issue was closed and marked as completed? Yes, with the help of screen reader programs, we have the ability to set one of 7 recent reactions to a specific message in the context menu. However, on the panel under the "More reactions" button, all possible reactions are labeled as "Telegram.ViewModels.Drawers.StickerViewModel". We cannot set reactions from the full list of possible reactions because we simply cannot obtain a designation and a normal name for an emoji reaction.
Perhaps you have fixed this issue and we should wait for its resolution in the new version of Unigram? The problem still persists in version 10.4.

**Comment by FrayxRulez** [comment]
@Dementor316 The problem is fixed in version 10.4.1.
